service/test/integration2_test.go

```diff
@@ -1048,8 +1048,16 @@ func TestSkipPrologue2(t *testing.T) {
 
 		callme3 := findLocationHelper(t, c, "main.callme3", false, 1, 0)[0]
 		callme3Z := findLocationHelper(t, c, "main.callme3:0", false, 1, 0)[0]
-		// callme3 does not have local variables therefore the first line of the function is immediately after the prologue
-		findLocationHelper(t, c, "callme.go:18", false, 1, callme3Z)
+		ver, _ := proc.ParseVersionString(runtime.Version())
```

Comment:
Second value is not an error, it's a bool indicating whether the version string is parsable. In particular we can't parse the version string of development versions of go (because they contain the sha-1 abbrev of HEAD), this condition is covered by the `ver.Major < 0` check.